docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -989,7 +989,7 @@ The following properties are _required_ unless a default value is available:
 | Property | Default value | Description |
 | :------- | :------------ | :---------- |
 | connector.class | N/A | Specifies the connector to use to connect Debezium to the database. For YugabyteDB, use `io.debezium.connector.yugabytedb.YugabyteDBConnector`. |
-| database.hostname | N/A | The IP address of the database host machine. For a distributed cluster, use the leader node's IP address. |
+| database.hostname | N/A | The IP address of the database host machine. For a distributed cluster, use the leader node's IP address.  Alternatively, a coma separated list of multiple host addresses along with corresponding ports can be specified, for ex: `ip1:port1,ip2:port2,ip3:port3`. This is useful for connection fail-over cases. |
```

Comment:
```suggestion
| database.hostname | N/A | The IP address of the database host machine. For a distributed cluster, use the leader node's IP address.  Alternatively, you can specify a comma-separated list of multiple host addresses along with corresponding ports ( for example,`ip1:port1,ip2:port2,ip3:port3`). This is useful for connection fail-over cases. |
```

Reply:
@Sumukh-Phalgaonkar - Is this applicable for 2.18 as well?

Reply:
Hi @aishwarya24 , I have back ported these changes to stable as well. Should I go ahead and merge the PR or will the docs team merge this?

Reply:
I don't see Aishwarya's fix, in either preview or stable, did you forget to commit it?

Reply:
@Sumukh-Phalgaonkar - Thanks . But I don't see my suggested changes in it. If you want, I can take it from here and make the changes to the PR.

Reply:
@aishwarya24  I had  Initially made the changes in the file `docs/content/**preview**/explore/change-data-capture/debezium-connector-yugabytedb.md`. For the Back port to stable I made the same changes to the file `docs/content/**stable**/explore/change-data-capture/debezium-connector-yugabytedb.md`. It would be great if you could take over the PR.

Reply:
I can merge it. Can you sign the agreement.